internal/rootcoord/quota_center.go

```diff
@@ -729,8 +714,20 @@ func (q *QuotaCenter) calculateReadRates() error {
 			databaseMetric[collection] += rm.Rate
 		}
 	}
+	return metricMap, collectionMetricMap
+}
+
+func (q *QuotaCenter) getLimit(metricMap map[string]float64,
```

Comment:
What you mean by `getLimit`?  Renaming it to a more specific function name would be better.